subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformInputArtifactIntegrationTest.groovy

```diff
@@ -74,7 +74,7 @@ class ArtifactTransformInputArtifactIntegrationTest extends AbstractDependencyRe
                 }
             }
 
-            abstract class MakeGreen implements TransformAction {
+            abstract class MakeGreen implements TransformAction<TransformParameters> {
```

Comment:
These should point to something like `TransformParameters.None` or something, no?

Reply:
I am using now `TransformParameters.None`.